streaming/src/main/scala/org/apache/spark/streaming/scheduler/rate/PIDRateEstimator.scala

```diff
@@ -39,13 +41,17 @@ package org.apache.spark.streaming.scheduler.rate
  *        of future errors, based on current rate of change. This value should be positive or 0.
  *        This term is not used very often, as it impacts stability of the system. The default
  *        value is 0.
+ * @param minRate what is the minimum rate that can be estimated.
+ *        This must be more than zero, so that the system always receives so data for rate
```

Comment:
nit: "greater than zero" sounds nicer. Also, "receives so data for rate" seems a typo, probably "some data"